src/complex/Filter/Actions/ImportH5ObjectPathsAction.cpp

```diff
@@ -48,15 +48,14 @@ Result<> ImportH5ObjectPathsAction::apply(DataStructure& dataStructure, Mode mod
   bool preflighting = (mode == Mode::Preflight);
 
   H5::FileReader fileReader(m_H5FilePath);
-  H5::ErrorType errorCode;
-  const DREAM3D::FileData fileData = DREAM3D::ReadFile(fileReader, errorCode, preflighting);
-  if(errorCode < 0)
+  const Result<DREAM3D::FileData> fileData = DREAM3D::ReadFile(fileReader, preflighting);
+  if(fileData.invalid())
   {
-    return {nonstd::make_unexpected(std::vector<Error>{Error{errorCode, "Failed to import a DataStructure from the target HDF5 file."}})};
+    return {nonstd::make_unexpected(std::vector<Error>{Error{fileData.errors().front().code, "Failed to import a DataStructure from the target HDF5 file."}})};
```

Comment:
The errors from importing can be returned here directly.

Reply:
Fixed by ce1be66a57370a6f61d9ce14d9d9ae9c095e6f1b